src/main/reactjs/src/components/clerkTranslator/ClerkTranslatorRegistry.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Paper, Grid, Divider, Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+import { useAppSelector } from 'configs/redux';
+import { useAppTranslation } from 'configs/i18n';
+import {
+  RegisterControls,
+  ListingFilters,
+} from 'components/clerkTranslator/ClerkTranslatorFilters';
+import { H1, H2, Text } from 'components/elements/Text';
+import { ClerkTranslatorListing } from 'components/clerkTranslator/ClerkTranslatorListing';
+import {
+  clerkTranslatorsSelector,
+  selectFilteredSelectedIds,
+} from 'redux/selectors/clerkTranslator';
+import { AppRoutes } from 'enums/app';
+
+const SendEmailButton = () => {
+  const { t } = useAppTranslation({ keyPrefix: 'akt.pages.clerkHomepage' });
+  const disabled = useAppSelector(selectFilteredSelectedIds).length === 0;
+
+  return (
+    <Button
+      data-testid="clerk-translator-registry__send-email-btn"
+      component={Link}
+      to={AppRoutes.ClerkSendEmailPage}
+      color="secondary"
+      variant="contained"
+      disabled={disabled}
+    >
+      {t('sendEmail')}
+    </Button>
+  );
+};
+
+export const ClerkTranslatorsRegistry = () => {
```

Comment:
Tässä jäin miettimään, että tarvitaanko clerktranslatorsregisterä ollenkaan. Lähinnä sen takia, että ClerkHomePage on melko tyhjä. Pitäisikö ClerkTranslatorsRegistry integroida kokonaan ClerkHomePageiin?

Reply:
Joo, vähän samaa itsekin olen pohtinut. Ehkä koko roskan voisi vain siirtää ClerkHomePagen alle.